src/components/Header/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import './Header.module.css';
+import React from 'react';
+import s from './Header.module.css';
+import { Link } from 'react-router-dom';
+import Search from '../Search/Search';
+
+const Header = () => {
+  return (
+    <header className={s.header}>
+      <div className={s.headerContainer}>
+        <div className={s.headerSection}>
+          <div className={s.headerLogo}>DYO</div>
+          <Search />
+        </div>
+        <div className={s.headerItems}>
+          <Link to={'/main'} className={s.headerItem}>
+            Main
+          </Link>
+          <Link to={'/aboutus'} className={s.headerItem}>
+            About Us
+          </Link>
+          <div className={s.headerItem}>Cards</div>
```

Comment:
const routeLinks = [{title: 'Main, to: '/main'}, {title: 'aboutus, to: '/aboutus'}]
...
{routeLinks.map( ...)}